test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1692,7 +1824,13 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		})
 
 		AfterEach(func() {
-			//Revert the profile
+			profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
```

Comment:
pay attention that this is for the  Network Stack Pinning context

Reply:
Yes I know. Not reverting if no changes is good in general.

Reply:
I like a lot this idea. Let's evaluate if we can make it more explicit.